public/controllers/agent/agents-preview.js

```diff
@@ -258,4 +263,18 @@ export class AgentsPreviewController {
       '_blank'
     );
   }
+
+
+  /**
+   * Returns the Wazuh version as x.y.z
+   */
+  async getWazuhVersion() {
+    try {
+      const data = await this.apiReq.request('GET', '/version', {}); 
+      const version = ((data.data || {}.data) || {}).data || {};
```

Comment:
1. Rename `const version` to `const result`.


2. Fix this declaration:
```js
((data.data || {}.data) || {}).data || {};
```

Should be:

```js
((data || {}).data || {}).data;
```